shakenfist/external_api/app.py

```diff
@@ -876,10 +856,23 @@ def delete(self, confirm=False, namespace=None):
             if instance['state'] == 'deleted':
                 continue
 
-            instance_from_db_virt = virt.from_db(instance['uuid'])
-            _delete_instance(instance['uuid'], instance_from_db_virt)
+            # If this instance is not on a node, just do the DB cleanup locally
```

Comment:
Maybe place this in a function to stop code repetition.

Reply:
I'm confused... This code isn't repeated?